### ISSUE TITLE
Screen Reader: volume control does not announce its state

### ISSUE THREAD

**Issue by miesgre** [polimediaupv/paella-core#296]
The volume control does not announce when the volume is muted or not muted. 
It also does not announce what level of volume it is at when the volume is changed. 
It should announce it's state. 

When using the keyboard, volume does announce it’s state. But it’s not announcing it with the screen reader.

@karendolan

**Comment by polimediaupv** [comment]
I'm sorry, but I'm afraid I have no idea how to implement this.

So far we have implemented all the accessibility features with the help of volunteer consultants. If you could give me more information on how I can implement and test this feature I would appreciate it.

**Comment by miesgre** [comment]
From @karendolan email:

The issue we had was the display=none hide the slider from the screen reader, so it could not be found when the video was playing. Using opacity = 0 kept the slider accessible to screen readers so it could be found. 

These are the changes for **es.upv.paella.volumeButtonPlugin.js**:

```diff
   showSideContainer() {
     if (!this.volumeAlwaysVisible) {
+      // this.sliderContainer.style.transition = 'scaleX(1) opacity 1s';
       this.sliderContainer.style.display = 'inline-block';
+      this.sliderContainer.style.opacity = '1';
+      this.sliderContainer.style.visibility = '100px';
+      this.sliderContainer.style.transition = 'all 1s';
     }
   }
 
   hideSideContainer() {
     if (!this.volumeAlwaysVisible) {
+      this.sliderContainer.style.opacity = '0';
+      this.sliderContainer.style.transition = 'all 1s';
+      this.sliderContainer.style.width = '0px';
-      this.sliderContainer.style.display = 'none';
     }
   }
```
